src/dune_rules/super_context.ml

```diff
@@ -354,6 +354,12 @@ let build_dir_is_vendored build_dir =
   | Some src_dir -> Dune_engine.Source_tree.is_vendored src_dir
   | None -> Memo.return false
 
+let add_vendored_flags ~ocaml_version flags =
```

Comment:
Maybe calling this function `with_vendored_flags` would be more coherent ?

Reply:
Sure!